app/models/course.rb

```diff
@@ -210,6 +210,10 @@ def new_draft_attributes
     joins(:provider).merge(Provider.where(provider_name: provider_name))
   end
 
+  scope :with_send, -> do
```

Comment:
Might be worth considering a recent related discussion regarding scopes
https://github.com/DFE-Digital/teacher-training-api/pull/1188#discussion_r382725562

Reply:
Should I remove the scope and do 
`scope = Course.where(is_send: true) if send_courses_filter?` in the service? 🤔